runtime/doc/news.txt

```diff
@@ -147,6 +147,11 @@ The following changes to existing APIs or features add new behavior.
   resulting in a nvim binary which only could be run headless or embedded
   in an external process. As of this version, TUI is always available.
 
+• The visual mode default-mappings ('*', '#') were changed to escape the
+  selected text.
+
+  See https://github.com/neovim/neovim/pull/21677.
```

Comment:
Though the change is valuable, we generally should not add robustness improvements or most bug fixes in news.txt. It's not a changelog, it's a high-level newsletter.

You can ignore the failing "news.txt" CI job, or rename the PR/commit to `fix(..)` which is more appropriate anyway.